### PR TITLE
Menuitem role has non-empty accessible name -- Remove outdated accsupport note

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Removes outdated accsupport note. Tested AT now behaves more conssistently 
Closes issue(s):

- closes #XXX (ADD ISSUE NUMBER HERE)

Need for Call for Review:
This will not require a Call for Review 

---

### **When creating PR:**

- [x] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [x] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [x] Add yourself (and co-authors) as "Assignees" for PR.
- [x] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [x] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [x ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by 0066135 — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
I don't think this needs a call for review @daniel-montalvo. Three approvals should be enough here IMO.

**Review by tombrunet — APPROVED** [review]

**Comment by daniel-montalvo** [comment]
Happy to merge once we get three approvals and if no one has concerns, @WilcoFiers

**Review by carlosapaduarte — APPROVED** [review]

**Pull request merged**
